tests/datasource/fluent/data_asset/test_sql_asset.py

```diff
@@ -68,23 +77,75 @@ def test_get_batch_identifiers_list__sort_descending(postgres_asset):
         batches[i]["year"] = year
 
 
+class FakeResult:
```

Comment:
Since we actually make a request to the database now to validate the batch definition column is valid, I now have to mock out the db interaction.